django/contrib/sessions/backends/cached_db.py

```diff
@@ -52,7 +57,12 @@ def exists(self, session_key):
 
     def save(self, must_create=False):
         super().save(must_create)
-        self._cache.set(self.cache_key, self._session, self.get_expiry_age())
+        try:
+            self._cache.set(self.cache_key, self._session, self.get_expiry_age())
+        except UpdateError:
```

Comment:
@shangxiao @sulabhkatila I'm not being able to understand why this `except UpdateError` is needed. I have commented out amd no test failing. I'm also trying to think of a proper test to cover for this code path, but I think it's unreachable: `UpdateError` could be raised when calling `super()` but not from any?/the cache backend.

What's your view on this?

Reply:
Thank you! I will push some cleanups and docs updates including the removal of this except block.

Reply:
Hi, it appears I'd added the ```except UpdateError``` without proper consideration at the time. I tested the code without the block, and, indeed, no tests are failing. I apologize for any confusion it may have caused. I'll make sure to be mindful of such changes in the future.

Reply:
> Thank you! I will push some cleanups and docs updates including the removal of this except block.

Thank you, nessita.